1.PyGMT_GMT_comparison.ipynb

```diff
@@ -0,0 +1,166 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# The PyGMT version:\n",
+    "import pygmt\n",
+    "\n",
+    "fig = pygmt.Figure()\n",
+    "fig.basemap(projection=\"N15c\", region=[0, 360, -90, 90], frame=True)\n",
+    "fig.coast(land=\"tan\", water=\"lightblue\")\n",
+    "fig.text(position=\"MC\", text=\"GMT\", font=\"80p,Helvetica-Bold,red@75\")\n",
```

Comment:
Hm. Maybe we can write in the PyGMT version the text "PyGMT" instead of "GMT"? Or do we want to have the figures / maps look completely identical?

```suggestion
    "fig.text(position=\"MC\", text=\"PyGMT\", font=\"80p,Helvetica-Bold,red@75\")\n",
```